galaxy.yml

```diff
@@ -12,7 +12,7 @@ tags:
   - vmware
   - virtualization
 dependencies:
-  cloud.common: ">=2.0.4,<3.0.0"
+  cloud.common: ">=2.0.4,<=3.0.0"
```

Comment:
This should either be `>=2.0.4,<4.0.0` or `>=3.0.0,<4.0.0`. I would go with the latter since we're doing a major version upgrade.

Reply:
> This should either be `>=2.0.4,<4.0.0` or `>=3.0.0,<4.0.0`. I would go with the latter since we're doing a major version upgrade.

I don't understand this. The dependencies of this collection don't have _anything_ to do with whatever future releases _you_ plan for `cloud.common`.

And: A dependency on `>=3.0.0` doesn't work because the latest release is [2.1.4](https://github.com/ansible-collections/cloud.common/releases/tag/2.1.4) at the moment. This will simply make this collection un-installable IMHO.

Your suggestion doesn't make any sense at all!

Once you've released `cloud.common` 3.0.0, the situation might change. But at the moment, requiring `cloud.common>=3.0.0` would break the collection.